builder/Dockerfile.ubuntu-1804

```diff
@@ -11,10 +11,15 @@ RUN set -x \
 
 RUN pip install awscli
 
+RUN apt-get install -y gcc make ruby ruby-dev wget
```

Comment:
include these in the `apt-get install` line above so that they are in the same layer

Reply:
Done